Packs/CommonPlaybooks/Playbooks/playbook-Search_And_Delete_Emails_-_Generic_-_v2.yml

```diff
@@ -392,22 +392,22 @@ tasks:
               complex:
                 root: modules
                 filters:
-                - - operator: isEqualString
```

Comment:
This playbook is not really supported anymore, but if you wish to fix it too, then you should also add the EWS fix